.github/.syncignore

```diff
@@ -1 +1,2 @@
 CODEOWNERS
+update-dependencies.yml
```

Comment:
You are correct.

That being said, other buildpacks without dependencies still do have this workflow, and it runs every hour and does nothing.

https://github.com/paketo-buildpacks/poetry-install/blob/main/.github/workflows/update-dependencies.yml

Is that adding to our overall actions workload? I'm comfortable removing it from here.

Reply:
I don't have strong opinions either way as to whether we add it to the `.syncignore` or not. Eventually it will disappear from github-config as part of the dependencies work, but that will take time. I'd prefer not to run extra actions that don't do anything, as it consumes resources unnecessarily, but on the other hand I worry that this `.syncignore` entry will get forgotten about.